docs/linux-requirements.md

```diff
@@ -0,0 +1,62 @@
+
+# Linux
+
+## Requirements to build native modules
```

Comment:
I think it would be helpful to explain **why** you might want to build native modules here.